README.md

```diff
@@ -37,13 +37,14 @@ There are two tools in this repository:
 Read more about this tool [here](doc/dependency-resolver.md).
 
 ```
-$ php bin/console oat:dependencies:resolve [--repository-name <repository name> | --extension-name <extension name>] [--main-branch <main repository branch>] [--dependency-branches <dependency branches>]
+$ php bin/console oat:dependencies:resolve [--repository-name <repository name> | --extension-name <extension name>] [--main-branch <main repository branch>] [--dependency-branches <dependency branches>] [--repositories]
 ```
 
 - `main repository name`: repository name, e.g. "oat-sa/extension-tao-testqti" of the repository to resolve.
 - `main extension name`: "manifest name", e.g. "taoQtiTest" of the extension to resolve.
 - `main repository branch`: the branch of the extension to be resolved.
 - `dependency branches`: desired branches to download include for each dependency. In the form of "extensionName1:branchName1,extensionName2:branchName2,...", e.g. "tao:develop,taoQtiItem:fix/tao-1234,generis:10.12.14". Branches for all non given extensions will default to "develop".
+- `repositories`: flag to indicate that composer repositories information must be included.
```

Comment:
I find it quite an important detail that for private repositories, ssh auth must be configured. Would you care documenting this behaviour too?